scripts/externalize_js.sh

```diff
@@ -45,18 +41,29 @@ if [[ ! $has_gnu_sed ]]; then
     should_continue
 fi
 
-has_local_changes=`git diff-index HEAD` && true
-if [[ $has_local_changes ]]
-then
-    abort "You have uncommitted changes in your working tree."
-fi
 
-branch=$(git rev-parse --abbrev-ref HEAD)
-if [[ $branch == 'master' ]]
-then
-    abort "You must not be on master to run this command."
+echo "Do you want this script to commit changes for you?"
```

Comment:
Can this just happen, rather than being an option? Not auto-committing will likely lead to a headache for the code reviewer.

Reply:
Sure, that works. Seemed like there might theoretically be times where I wouldn't want it to do so immediately, but it's also true that it'd be a problem for the reviewer